### DISCUSSION TITLE
Is it possible to update firmware without impacting on the hack?

### DISCUSSION THREAD

**Discussion by hiagocosta** [roleoroleo/yi-hack-Allwinner-v2#728] [General]
I have on my yi home app, a notification for update the cam,

since i'm using the hack for allwinner-v2, can I update the firmware?
and I figured that if i'm using yi cloud the clock on the top corner left is right on time, but when I turn off, it changes for 6 hours after my own TZ (BRT set on the yi hack config)

here's my actual configuration in the screenshot below:

![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/22087481/103bf3da-fb7d-4999-a5fe-65c13a54503d)

**Comment by hiagocosta** [comment]
heres the notif.:
![WhatsApp Image 2023-10-21 at 15 40 48](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/22087481/81379bf8-5bab-4851-91fd-8a1d886d4ffd)

**Comment by roleoroleo** [comment]
I don't know.
Unfortunately it depends on the content of the update.

**Comment by hiagocosta** [comment]
I saw that this update os for 12 version, and by The table of working
devices, it should work

Em dom., 22 de out. de 2023 11:00, roleo ***@***.***>
escreveu:

**Comment by hiagocosta** [comment]
I update the camera, and it working!
I dont need to hack it again, but I resete the config and load the backup I made before, and it seems that is faster now!
o got some stucks in http cam before, that I could manage by unplug the camera and plug it again, and now I don't need it anymore! :D

**Comment by hiagocosta** [comment]
but bro, can you help me to include nanotts to work in my cam?
I added the files just like that:

extracted the tgz file on windows explorer, and send via ftp the files directly indo the folders:
nanotts > /tmp/sd/yi-hack/bin
folder pico of usr > in /tmp/sd/yi-hack/usr/share

is it correct?
can I test using GET method?
tried with postman, but I can't include the json in it, or you sugest to use another app for this?

sorry, not an english native and not so expert in web development about how test json method

**Reply by roleoroleo** [reply]
Use the test page:
http://IP_OF_YOUR_CAM/?page=speak
